charts/newrelic-infra-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+rules:
+  - apiGroups: [""]
+    resources:
+      - "secrets"
+    verbs: ["get", "update", "patch"]
+    resourceNames: [{{ template "newrelic-infra-operator.fullname" . }}-config]
+  - apiGroups: [""]
+    resources:
+      - "secrets"
+    verbs: ["create"]
```

Comment:
Can we use `resourceNames` as well?

Reply:
Sadly it doesn't seem to be possible for the `create` verb, as we discussed earlier :(